### ISSUE TITLE
installing with npm on windows fails with `Error: Could not find "dep-tree" in archive`

### ISSUE THREAD

**Issue by MatthijsKok** [gabotechs/dep-tree#52]
Hi, your library looks stunning, and especially the "check" dependency linting makes me want to try it out.
Unfortunately I'm having problems with installing it.

Trying to install it with `npm` (or `pnpm`) does not work, and produces the following error:
```
C:\Users\mkok>npm install --global @dep-tree/cli
npm ERR! code 1
npm ERR! path C:\Users\mkok\AppData\Roaming\nvm\v20.10.0\node_modules\@dep-tree\cli
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node install.js
npm ERR! Downloading dep-tree release from https://github.com/gabotechs/dep-tree/releases/download/v0.17.2/dep-tree_0.17.2_windows_amd64.tar.gz
npm ERR! Error: Could not find "dep-tree" in archive
npm ERR!     at extractFileFromTarGzip (C:\Users\mkok\AppData\Roaming\nvm\v20.10.0\node_modules\@dep-tree\cli\install.js:77:11)
npm ERR!     at install (C:\Users\mkok\AppData\Roaming\nvm\v20.10.0\node_modules\@dep-tree\cli\install.js:90:17)
npm ERR!     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

npm ERR! A complete log of this run can be found in: C:\Users\mkok\AppData\Local\npm-cache\_logs\2023-12-30T21_47_57_395Z-debug-0.log
```
The debug log can be found here
[2023-12-30T21_47_57_395Z-debug-0.log](https://github.com/gabotechs/dep-tree/files/13799243/2023-12-30T21_47_57_395Z-debug-0.log)

I have tried multiple older versions of `dep-tree`, they all produce the same error.  

Here are some specs that might be relevant:
>OS Platform: Windows 11 Enterprise
>OS Version: 10.0.22631 Build 22631
>node version: v20.10.0
>npm version: 10.2.3

Please let me know if you need any more information, or if I can assist with further debugging.

**Comment by gabotechs** [comment]
Hi @MatthijsKok! thanks for reporting.

I was able to replicate your issue, I'm currently working on a fix for https://github.com/gabotechs/node-dep-tree/pull/3.

**Comment by gabotechs** [comment]
The fix is out already under version `0.17.4`, if for some reason it still does not work for you feel free to reopen

**Comment by MatthijsKok** [comment]
The latest `0.17.4` release installs correctly and works.
Thank you for the quick fix!
